spec/facets/DatasetProviderDatasetFacet.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema" : "https://json-schema.org/draft/2020-12/schema",
+  "$id" : "https://openlineage.io/spec/facets/1-0-0/DatasetProviderDatasetFacet.json",
+  "$defs" : {
+    "DatasetProviderDatasetFacet": {
+      "allOf": [
+        {
+          "$ref": "https://openlineage.io/spec/1-0-2/OpenLineage.json#/$defs/DatasetFacet"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "datasetProvider": {
+              "description": "Dataset provider like iceberg, delta-lake, etc.",
+              "type": "string"
+            },
+            "datasetFormat": {
```

Comment:
Similar comment: we should document known formats

Reply:
Add to https://github.com/OpenLineage/OpenLineage/blob/main/spec/Naming.md perhaps?

Reply:
I think the description here should be more directive:
It is "iceberg" or "delta"
not "like iceberg or delta"